docs/plugins/pluggable_rule_engine.md

```diff
@@ -187,24 +187,29 @@ delay("hints") {
     - `dd.hh:mm:ss` - 2-digit integers representing days, hours, minutes, and seconds, respectively. Most significant values can be omitted (e.g. 20:40 means mm:ss)
     - `YYYY-MM-DD.hh:mm:ss` - Least significant values can be omitted (e.g. 2015-07-29.12 means noon of July 29, 2015)
   - `PLUSET` - Relative Execution Time - Relative to current time when the delayed execution should be performed.
+  - `PRIORITY` - Scheduled Execution Order - Controls the order at which rules are scheduled for execution.
+  - `PRIORITY` - Scheduled Execution Order - Controls the execution order of scheduled rules.
+  - `PRIORITY` - Scheduled Execution Order - Controls when the rule will be scheduled for execution in relation to sibling rules.
+  - `PRIORITY` - Scheduled Execution Order - A hint to the delay server on how to schedule rules for execution.
+    - A valid priority value is an integer in the range [1, 9] where 1 represents the lowest level and 9 represents the highest level.
```

Comment:
This would be paired with one of the statements above.

Reply:
I like the first...  building on it...

```
  - `PRIORITY` - Scheduled Execution Priority - Controls the execution order of scheduled rules. Valid priority values are integers in the range [1, 9] where 9 represents the highest priority (executed earlier).  Default priority is 5.
```

Reply:
Ok. I'll add those changes and let you see how it is rendered.

Reply:
Did you see the very first PRIORITY statement? This code view doesn't show all of the choices.

Reply:
yes, i saw all four.

Reply:
Ok. Preparing to update and add the #

Reply:
done